tasks/lint.js

```diff
@@ -16,7 +16,18 @@ const config = require('../config')();
  * are detected.
  */
 function lintScss() {
-  return src(config.pipelines.styles.src)
+  let fullSrc = Array.isArray(config.pipelines.styles.src) ?
```

Comment:
I have full faith that this all works, but can we add some comments to this to help future us/current Adam who is boggled by all of this JS. 😂

Reply:
Gulp accepts either a string or an array of strings for its source, so Calliope also accepts either option. This applies both to the `src` key and the `watch` key. The issue I was trying to solve with this bit of code here is handling each case, but your request for clarification by way of a comment made me realize it’s a bit too difficult to follow.

So I did you one better: I decided to use [`lodash.flattendeep`](https://npmjs.com/package/lodash.flattendeep) to make this code easier to read and let that library handle the complexity of taking potentially nested arrays and flatten them out into a single-level array. See aee89b0.

Reply:
I tested that locally and it works just fine.

Reply:
💥

Reply:
In hindsight it wasn't that confusing before. Code review at the end of the afternoon during peak fried brain is not a good idea though. 😬 

I like your new solution even better though!

Reply:
I like code that is easy on the fried brain. 😜